.pre-commit-config.yaml

```diff
@@ -0,0 +1,52 @@
+# for when we add pre-push hooks, so users only need to run pre-commit install
+default_install_hook_types: [pre-commit, pre-push]
+exclude: "^mage_ai/data_preparation/templates|^mage_ai/orchestration/db/migrations/versions"
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: trailing-whitespace
+        types: [python]
+    -   id: end-of-file-fixer
+        types: [python]
+    -   id: check-yaml
+        types: [python]
```

Comment:
check-yaml should be applied to yaml, right?

Reply:
I switched it from python to YAML.